extensions/widgets/layout/src/components/index.tsx

```diff
@@ -268,7 +268,7 @@ const Layout: React.FC<unknown> = () => {
                 </Stack>
               </Stack>
             </Stack>
-            <Stack className="fixed bottom-2 lg:(w-[21rem])">
+            <Stack className="fixed bottom-2 lg:(w-[21.125rem])">
```

Comment:
this value is weird, 21.125rem :)) ...i see what you are trying to fix but did you look for another way to fix it ?